### PR TITLE
add python runtime option in GCF rule

### PR DESCRIPTION
**Pull request by zweihan**
Hi, 

Appreciate your repo! 

I have a need for this and has added an option to specify runtime.

### CODE REVIEW

**Comment by zweihan** [comment]
I also converted the Fish script to Bash for my own use. If you are interested, I can do a PR for that as well.

File: infra/serverless/gcf_rules.bzl [hunk 1]
```diff
@@ -6,6 +6,7 @@ MEMORY_VALUES = [
   128, 256, 512, 1024, 2048, 4096,
 ]
 MAX_TIMEOUT = 540
+PYTHON_RUNTIME = ["python37", "python38", "python39"]
```
Comment (weisi) [inline infra/serverless/gcf_rules.bzl]: `PYTHON_RUNTIMES`

**Review by weisi — CHANGES_REQUESTED** [review]
Thanks! I'd appreciate if you rename `PYTHON_RUNTIME` to `PYTHON_RUNTIMES` - and then I will merge this PR.

**Comment by weisi** [comment]
Regarding the fish to sh rewrite - I appreciate that, though I'm not sure if I want to have sh scripts in this repo. I understand that using sh is more universal, however on the other hand I feel fish scripting is easier to read (which is why I really want to use it). Do you think it would make sense if I link to your repo from this repo's README file, and let people know if they prefer using sh, they can import your repo instead? Thanks!

**Comment by zweihan** [comment]
Updated to PYTHON_RUNTIMES

**Comment by zweihan** [comment]
> Regarding the fish to sh rewrite - I appreciate that, though I'm not sure if I want to have sh scripts in this repo. I understand that using sh is more universal, however on the other hand I feel fish scripting is easier to read (which is why I really want to use it). Do you think it would make sense if I link to your repo from this repo's README file, and let people know if they prefer using sh, they can import your repo instead? Thanks!

No worries! It's just Bash is more commonly used and it doesn't make sense to have to install a new Shell in order to use the bazel rules. I can keep the fork. :)

**Pull request merged**